src/main/java/io/confluent/connect/hdfs/wal/FSWAL.java

```diff
@@ -35,15 +35,13 @@
 public class FSWAL implements WAL {
 
   private static final Logger log = LoggerFactory.getLogger(FSWAL.class);
-  private static final String leaseException =
-      "org.apache.hadoop.hdfs.protocol.AlreadyBeingCreatedException";
-  private static final long MAX_SLEEP_INTERVAL_MS = 16000L;
 
   private WALFile.Writer writer = null;
   private WALFile.Reader reader = null;
   private String logFile = null;
   private HdfsSinkConnectorConfig conf = null;
   private HdfsStorage storage = null;
+  private long sleepIntervalMs = WALConstants.INITIAL_SLEEP_INTERVAL_MS;
```

Comment:
Why is this one upgraded from local variable to member field variable? The behavior definitely changes on repeated calls on the same object. Notice that this variable is mutated iteratively as part of the `while` loop in `acquiredLease`. Even if it happens and there's only one call of `acquireLease` per `FSWAL` object I think it's a better practice to keep it local if that's what we intend to do here.